tools/SourceKit/include/SourceKit/Core/LangSupport.h

```diff
@@ -286,7 +286,7 @@ struct CursorInfoData {
   StringRef ModuleInterfaceName;
   /// This is an (offset,length) pair.
   /// It is set only if the declaration has a source location.
-  llvm::Optional<std::pair<unsigned, unsigned>> DeclarationLoc;
+  llvm::Optional<std::pair<unsigned, unsigned>> DeclarationLoc = None;
```

Comment:
Huh, I wonder why Optional's default constructor is explicit. (It looks like it always has been. `std::optional`'s isn't.

Reply:
I don't know the reason for why the requirements on the constructor is different between the two implementations.  But, this is something which does bite MSVC from time to time too.  It is rather unfortunate.